app/models/lookup_keys/lookup_key.rb

```diff
@@ -57,7 +57,7 @@ def self.inherited(child)
   alias_attribute :override_values_count, :lookup_values_count
   alias_attribute :override_values, :lookup_values
   alias_attribute :override_value_ids, :lookup_value_ids
-  attr_accessible :lookup_values_attributes, :is_param, :id, :variable, :key
+  attr_accessible :lookup_values_attributes, :is_param, :id, :variable, :key, :puppetclass_id
```

Comment:
I've put this one on variable_lookup_key , otherwise puppetclass_id might not even be an attribute